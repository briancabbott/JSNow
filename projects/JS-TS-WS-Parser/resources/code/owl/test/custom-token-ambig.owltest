---
.token integer "1" "2342"
.token imaginary "1.0j"
.token other

input = integer integer imaginary other other x*
x = ''
