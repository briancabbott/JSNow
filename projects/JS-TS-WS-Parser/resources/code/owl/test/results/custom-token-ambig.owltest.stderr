error: this grammar is ambiguous

. 1 2342 1.0j other other 

  can be parsed in two different ways: as

. 1 2342 1.0j other other 
  input------------------ 

  or as

. 1 2342 1.0j other other  
  |                      x 
  input------------------- 

